pkg/applicationserver/applicationserver_test.go

```diff
@@ -197,6 +200,14 @@ hardware_versions:
 	defer webhooksRedisClient.Close()
 	webhookRegistry := iowebredis.WebhookRegistry{Redis: webhooksRedisClient}
 
+	natsServer := nats_server.RunDefaultServer()
+	defer natsServer.Shutdown()
+	time.Sleep(Timeout)
+	err = os.Setenv("NATS_SERVER_URL", "127.0.0.1")
```

Comment:
What is this for?

Reply:
The `natspubsub` unfortunately has this very antique way of setting the server to which the client connects - a single environment variable. See [natspubsub](https://godoc.org/gocloud.dev/pubsub/natspubsub)